src/routes/index.js

```diff
@@ -2,10 +2,13 @@ const express = require("express");
 
 const router = express.Router();
 
-const userRoute = require("./user.route");
+const authRoute = require("./auth.route");
 const projectRoute = require("./project.route");
 
-router.use("/users", userRoute);
+const setResponseHeader = require("./middleware/setResponseHeader");
+
+router.use(setResponseHeader);
+router.use("/auth", authRoute);
 router.use("/projects", projectRoute);
```

Comment:
```suggestion
router.use("/projects", verifyToken, projectRoute);
```
권한체크를 여기서 토큰검사로 하면 어떨까요

Reply:
추후 논의 필요한 사항

Reply:
저도 전에 이런 방식을 사용했었어요! 모든 요청을 처리하기 전에 token을 검증하는 로직이 필요하다면 여기서 한 번에 하는 게 깔끔할 것 같습니다!